### ISSUE TITLE
Implement Download the file functionality

### ISSUE THREAD

**Issue by hypermystic** [hypermystic/bookit#12]
Where is download button

1: Setup the project.
2: Login into the app
3: Create a Book
4: Add a chapter
5: Now a download file button appears on the right side

**Comment by absenteerealitycheck** [comment]
Hi @hypermystic !
Can I work on this issue?

**Comment by hypermystic** [comment]
yeah sure ...assigned and if you face any problem in setting up the project..please read the instructions in issue "auto redirect to https://todo-7e79c.web.app/"

**Comment by hypermystic** [comment]
Currently ...the button is present but it is not doing anything
